tests/test_issue_retrieval.py

```diff
@@ -0,0 +1,57 @@
+from src import issue_retrieval
+from json_interaction import json_handler
+from github import Github
+
+# As of the current state, these tests require a token to function
+#
+# This requirement will be amended within the week after successful team
+# collaboration provided an alternative test procedure is possible
+#
+# Remove your token from this file after demonstration
+token = "9fd3989585322606a5107412bb9c2ffa82ad25ee"
+repository_name = "GatorCogitate/cogitate_tool"
+contributor_data = json_handler.get_dict_from_json_file("contributor_data")
```

Comment:
Is this token gonna be taken out at any point? Or is this the account/token you said earlier created just for testing

Reply:
I forgot to pull it in one of the commits. I was just using it for testing, and generated a new one regardless.

Reply:
It should say "REDACTED" again